packages/posthog/package.json

```diff
@@ -77,7 +77,7 @@
   },
   "peerDependencies": {
     "@capacitor/core": ">=7.0.0",
-    "posthog-js": "^1.160.3"
+    "posthog-js": "^1.265.1"
```

Comment:
Revert this change. Otherwise, we cannot merge it before the next major release since this would be a breaking change.